packages/etcd/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.0"
+  changes:
+    - description: Add `ssl` configuration option for metricsets
+      type: enhancement
+      link: https://github.com/elastic/integrations/issues/9408
```

```suggestion
      link: https://github.com/elastic/integrations/pull/9414
```
Any reason to include the issue link instead of PR?

---

No, not really. Was not sure about the convention.